# Cloud Label
Header header

uint8[] label #  the label information for each point cloud
bool[] isHeightValid
float32[] height